assets/icons.js

```diff
@@ -454,12 +454,12 @@ export const BurgerIcon = ({ color = 'black' }) => (
   </svg>
 );
 
-export const BlackLogo = () => (
+export const LogoIcon = ({ color = 'black' }) => (
```

Comment:
you should create a new branch for this issue also. Do NOT include changes from another issue branch. It should include only changes for this issue